database/sql/src/main/scala/cromwell/database/sql/CallCachingSqlDatabase.scala

```diff
@@ -15,5 +16,5 @@ trait CallCachingSqlDatabase {
                       (implicit ec: ExecutionContext): Future[Option[CallCachingJoin]]
 
   def invalidateCall(callCachingEntryId: Int)
-                    (implicit ec: ExecutionContext): Future[Unit]
+                    (implicit ec: ExecutionContext): Future[Option[CallCachingEntry]]
```

Comment:
They didn't fit the shape of our problem.  We have very different handling of `Success(None)` versus `Failure`.